docs/geedocs/docsrc/BUILD_DOCS.md

```diff
@@ -6,14 +6,12 @@ source files and then generate HTML using Sphinx.
 
 ## Prerequisites
 
-To build the documentation, you need to install Sphinx:
+To build the documentation, the following packages (and versions) need to be installed :
 
-```bash
-sudo apt install python-pip
-sudo pip install sphinx==1.7.5
-sudo pip install GitPython
-sudo pip install sphinxprettysearchresults
-```
+python 2.7 (required)
```

Comment:
required is implied by your previous statement. (Sorry I missed that earlier...)